README.md

```diff
@@ -119,6 +119,12 @@ Check it out
 you can also
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/DeepRegNet/DeepReg/blob/main/docs/Intro_to_Medical_Image_Registration.ipynb)
 
+## Introductory Talk
+
+We presented (the road to) DeepReg at the Center for Medical Imaging Computing (CMIC)
```

Comment:
Incredibly nit-picky but it's Centre for Medical Imaging, not Center

Reply:
Fixing, fair enough!